.github/actions/deb-delivery/action.yml

```
@@ -25,6 +25,9 @@ inputs:
   stability:
     description: "The package stability (stable, testing, unstable)"
     required: true
+  arti_token:
```

IMHO it is more meaningful with full name
```suggestion
  artifactory_token:
```